frontend/libs/studio-components/src/components/StudioTextfield/StudioTextfield.test.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import type { StudioTextfieldProps } from './StudioTextfield';
+import { StudioTextfield } from './StudioTextfield';
+import { StudioTextarea } from '../StudioTextarea';
+import userEvent from '@testing-library/user-event';
+import type { RefObject } from 'react';
+
+describe('StudioTextfield', () => {
+  it('Renders a text field', () => {
+    renderTextfield();
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+  });
+
+  it('Renders with given label', () => {
+    const label = 'test-label';
+    renderTextfield({ label });
+    expect(screen.getByRole('textbox', { name: label })).toBeInTheDocument();
+  });
+
+  it('Renders with given label when there is an asterisk', () => {
+    const label = 'test-label';
+    renderTextfield({ label, withAsterisk: true });
+    expect(screen.getByRole('textbox', { name: label })).toBeInTheDocument();
+  });
```

Comment:
Any reason that the label is only tested for the textField and not TextArea?

Reply:
Only that this functionality comes from the shared hook, so if it works here, it also works on `StudioTextarea`. The tests that already exist on `StudioTextarea` should be more than sufficient to verify that the functions from the hook are applied.

Reply:
I have added the same tests to the textarea anyways.